applications/nrf_desktop/README.rst

```diff
@@ -1556,6 +1556,8 @@ The B0 bootloader requires the following options enabled:
 * :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
```

Comment:
In commit message `applications: ...`

Reply:
We also use `Jira: NCSDK-XXXX` instead of `Ref: NCSDK-XXXX`. Maybe we could align here?